utils/profile-actions.js

```diff
@@ -28,7 +29,8 @@ export function getRandomEmail() {
 }
 
 export function getSecondPurchaseEmail() {
-  return 'second-purchase-5@mailinator.com'
+  const NUMBER = between(10, 110)
+  return `second-purchase-${NUMBER}@mailinator.com`
```

Comment:
Adding a commentary looks valuable to me, as this random behavior might be weird for someone that doesn't have our context.

I've also added a nitpick regarding the variable name, because I believe that `index` is slightly more descriptive than `number`.

Does that make sense to you?

```suggestion
  // there is a known issue that the same account can't make two or more
  // purchases at the same time, so in order to use test parallelization,
  // we must use different accounts each time.
  const index = between(10, 110)
  return `second-purchase-${index}@mailinator.com`
```

Reply:
Yes, make total sense. I was supposed to add this comments, but I didn't. And `index` is better.
I already made those changes in my last commit, so I'm resolving this conversation... (I don't know this is the right to do).